src/components/reportPage/ReportPage.jsx

```diff
@@ -24,6 +27,33 @@ const ReportPage = (props) => {
   const [colourScheme, setColourScheme] = useState(inferno256);
   const [searchTerm, setSearchString] = useState("");
 
+  const [filterType, setFilterType] = useState("All");
+  const [videoFilterType, setVideoFilterType] = useState("0");
+
+  const [displayValue, getValue] = useState([]);
+  const animatedComponents = makeAnimated();
+
+  const options = [
+    { value: "5", label: "Card flipping" },
+    { value: "Component occlusion", label: "Component Occlusion" },
+    { value: "Missing image", label: "Missing Image" },
+    { value: "8", label: "Missing elevation" },
+    { value: "2", label: "Missing scrim" },
+    { value: "9", label: "Modal sheet missing scrim" },
+    { value: "6", label: "Moving cards behind other cards" },
+    { value: "1", label: "No defect" },
+    { value: "Null value", label: "Null Value" },
+    { value: "3", label: "Snackbar blocking bottom navigation" },
+    { value: "4", label: "Stacking multiple banners" },
+    { value: "7", label: "Stacking multiple snackbars" },
+  ];
```

Comment:
Could you order the video defects by the value please? And seperate the image and video defects, something like this:
```
const videoDefects = [
{ value: "1", label: "No defect" },
{ value: "2", label: "Missing scrim" },
...
];

const imageDefects = [
{ value: "Component occlusion", label: "Component Occlusion" },
...
];

const allDefects = [...videoDefects, ...imageDefects];
```

Reply:
Just noticed, "1" is being used as no defects for images. There is a "1" for video results though which is missing. Ig you have a look at Repository.js there is a mapping for the video results
![image](https://user-images.githubusercontent.com/53201250/132931976-706d2662-8a49-4b2a-a508-5426f0009e9e.png)